src/react-components/home-root.js

```diff
@@ -203,6 +203,9 @@ class HomeRoot extends Component {
                       <FormattedMessage id="editor-name" />
                     </a>
                   </IfFeature>
+                  <a href="https://hubs.mozilla.com/docs" rel="noreferrer noopener">
+                    <FormattedMessage id="home.docs_link" />
+                  </a>
```

Comment:
We should probably make this configurable for cloud users. Either disable the link by default and let them turn it on, or allow them to configure the link. You just need to add the corresponding configs to `src/schema.toml`

Reply:
See the existing `features.show_community_link` and `links.community` configs as examples

Reply:
Not sure if this is the same thing (having a difficult time tracking down where this configuration lives), but this may be how the `HelpDialog` is configured: https://github.com/mozilla/hubs/blob/master/src/react-components/help-dialog.js#L20

Reply:
`configs.link("features", messages["help.docs_url"])` means "take the configuration from the admin panel for the `links.features` config, or if that doesn't exist, use the `help.docs_url` string from the `translations.data.json` file"